homeassistant/components/roku/media_player.py

```diff
@@ -234,6 +255,19 @@ def source_list(self) -> list:
         """List of available input sources."""
         return ["Home"] + sorted(app.name for app in self.coordinator.data.apps)
 
+    @roku_exception_handler
+    async def launch(self, app_id: str, content_id: str, media_type: str):
+        """Launch application with content deeplink."""
+        params = {}
+
+        if content_id is not None:
+            params["contentID"] = content_id
+
+        if media_type is not None:
+            params["MediaType"] = media_type
+
+        await self.coordinator.roku.launch(app_id, params)
+
     @roku_exception_handler
     async def search(self, keyword):
```

Comment:
Maybe change this one as well to `async_search`. But thats existing so maybe a followup PR?

Reply:
noted will do in followup